src/intra_edge.rs

```diff
@@ -298,7 +294,10 @@ impl<const SB128: bool, const N_BRANCH: usize, const N_TIP: usize>
                 branch.split[n as usize] = child_branch;
                 (self, indices) = self.init_mode_node(
                     child_branch,
-                    bl + 1,
+                    match bl.decrease() {
+                        None => panic!("BlockLevel::BL_8X8 should never make it here"),
+                        Some(next_bl) => next_bl,
+                    },
```

Comment:
```suggestion
                    bl.decrease().expect("BlockLevel::BL_8X8 should never make it here"),
```

Reply:
I think expect isn't a `const fn`.

Reply:
yeah, in fact even `unreachable!` isn't allowed in const contexts which was surprising to me (intuitively it's just some pretty formatting around a `panic!`)